.claude/audit/scripts/census.py

```
@@ -0,0 +1,117 @@
+
+
+def is_infra(name):
+    if name in INFRA_NAMES:
+        return True
+    if name.startswith("GetExternFunction"):
+        return True
+    if "." in name:  # interface-explicit re-implementations (dup of __ form)
+        return True
+    if not name.startswith("__"):
+        return True
+    return False
```

Ruff (0.15.14)

[warning] Return the condition `bool(not name.startswith("__"))` directly

Replace with `return bool(not name.startswith("__"))`

(SIM103)